seungjong/routes/index.js

```diff
@@ -0,0 +1,14 @@
+// routes/index.js
+
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+const postRouter = require('./postRouter');
+const likeRouter = require('./likeRouter');
+
+router.use("/users", userRouter.router);
+router.use("/posts", postRouter.router);
+router.use("/likes", likeRouter.router);
```

Comment:
```suggestion
const express = require("express");

const userRouter = require("./userRouter");
const postRouter = require('./postRouter');
const likeRouter = require('./likeRouter');

const router = express.Router();

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
router.use("/likes", likeRouter.router);
```
1. 주석처리된 코드는 지워주세요.
2. 코드 작성은 필요한 모듈들을 불러오고 작성하는 순서로 작성되는것이 관례입니다.